backend/push.js

```diff
@@ -119,7 +119,7 @@ export const subscriptionInfoWrapper = (subcriptionId: string, subscriptionInfo:
         let salt
         let uaPublic
 
-        return function (this: Object) {
+        return function (/* this: Object */) {
```

Comment:
I had to comment out this annotation, despite it not causing any trouble before. Meanwhile, neither flow nor flow-remove-types have been updated (at least not directly, but maybe their subdep tree has).